Gems/LmbrCentral/Code/Source/Shape/ReferenceShapeComponent.cpp

```diff
@@ -27,7 +27,7 @@ namespace Vegetation
             if (edit)
             {
                 edit->Class<ReferenceShapeConfig>(
-                    "Vegetation Reference Shape", "")
+                    "Reference Shape", "")
```

Comment:
I think this should be "Shape Reference"?

Reply:
Fixed